docs/prow/prow-installation-on-forks.md

```diff
@@ -69,7 +69,17 @@ For the purpose of the installation, you must have a set of service accounts and
  - **kyma-bot-npm-token** which is a token for publishing npm packages.
  - **sap-slack-bot-token** which is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).
  - **sa-kyma-artifacts** service account with the role that allows the account to store objects in a bucket.
-    
+
+3. Create a Secret for the workload cluster in the Prow Cluster:
+
+    ```bash
+        export PROJECT={project-name}
+        export CLUSTER_NAME={cluster-name} ###  Use a different name as that of the Prow cluster
```

Comment:
TBH I don't understand it, you write that this secret should be created in the Prow Cluster and then cluster-name should be different, I am confused ;)

Reply:
Changed to WORKLOAD_CLUSTER_NAME